test/hotspot/jtreg/ProblemList-Virtual.txt

```diff
@@ -127,3 +128,15 @@ vmTestbase/nsk/jdi/stress/serial/forceEarlyReturn002/TestDescription.java 828541
 # of this no vthreads can complete their reads, and the test times out as a result.
 
 vmTestbase/nsk/jdi/VMOutOfMemoryException/VMOutOfMemoryException001/VMOutOfMemoryException001.java 8285417 windows-all
+
+##########
+## Tests incompatible with  with virtual test thread factory
+## There is no goal to run all test with virtual test thread factory
+## So any test migth be added as incompatible, the
```

Comment:
A typo at line 133: "with with". Dot is missed at the end.
It seems, both statements at 134-135 are incomplete.
First misses dot at the end, second ended with "the" and has no dot as well.